docs/SendGrid-batchsource.md

```diff
@@ -0,0 +1,52 @@
+# SendGrid batch source
+
+Description
+-----------
+This plugin used to query SendGrid v3 API.
```

Comment:
General information for user. Does not matter which API etc.

Reply:
Plugin fetches analytics data from SendGrid about Marketing Campaigns Fields, Stats Fields,  Suppressions Fields ... and more details regarding this.